manifests/init.pp

```diff
@@ -60,6 +60,10 @@
 #
 # [*manage_service*]
 #   Whether or not to manage the state of the Marathon service with Puppet.
+#
+# [*service*]
+#   Service to notify when there is a config change. If manage_service is true
+#   then this should not be set.
```

Comment:
We notify `Service['marathon']` when managing the service. A `notify =>` in puppet also `require`s the resource. So the config will only be added after the service is defined and the service will be notified when those config files change. Does that answer your question?